src/Backups/BackupCoordinationStageSync.cpp

```diff
@@ -1,6 +1,7 @@
 #include <Backups/BackupCoordinationStageSync.h>
 #include <Common/Exception.h>
 #include <Common/ZooKeeper/KeeperException.h>
+#include <Common/ZooKeeper/ZooKeeperWithFaultInjection.h>
```

Comment:
If you're not going to use zk client with fault-injection, then the header should not be necessary here. But it is worth using it.